### PR TITLE
QR code

### PR DESCRIPTION
**Pull request by Mike-3**
common supported image types
same qr code border for text export

### CODE REVIEW

File: lib/logic/tools/images_and_files/qr_code.dart [hunk 1]
```diff
@@ -1,12 +1,53 @@
+import 'dart:async';
 import 'dart:typed_data';
+import 'package:flutter/cupertino.dart';
```
Comment (S-Man42) [inline lib/logic/tools/images_and_files/qr_code.dart:3]: remove the cupertino import. Always use material instead

File: assets/i18n/en.json [hunk 2]
```diff
@@ -2547,6 +2547,8 @@
   "projectiles_energy": "Kinetic Energy",
   "projectiles_mass": "Mass",
   "projectiles_velocity": "Velocity",
+  "qr_code_title": "QR code/ Barcode",
```
Comment (S-Man42) [inline assets/i18n/en.json:2550]: QR Code/Barcode

File: assets/i18n/de.json [hunk 3]
```diff
@@ -2491,6 +2491,8 @@
   "projectiles_energy": "Kin. Energie",
   "projectiles_mass": "Masse",
   "projectiles_velocity": "Geschwindigkeit",
+  "qr_code_title": "QR Code/ Barcode",
```
Comment (S-Man42) [inline assets/i18n/de.json:2494]: QR Code/Barcode

File: assets/i18n/de.json [hunk 4]
```diff
@@ -2491,6 +2491,8 @@
...
+  "qr_code_description": "QR Code erkennen",
```
Comment (S-Man42) [inline assets/i18n/de.json:2495]: QR und Barcodes aus Bilddateien erkennen

File: assets/i18n/en.json [hunk 5]
```diff
@@ -2547,6 +2547,8 @@
...
+  "qr_code_description": "Recognize QR code ",
```
Comment (S-Man42) [inline assets/i18n/en.json:2551]: Recognize QR and barcodes from image files

Comment (Mike-3) [reply assets/i18n/en.json]: all changed

**Pull request merged**